projects/plugins/jetpack/extensions/shared/memberships/settings.scss

```diff
@@ -12,6 +12,12 @@
 	text-decoration: none;
 }
 
+.jetpack-newsletter-access-radio-buttons {
+	.components-radio-control__label {
```

Comment:
Do we need the core class here (which could change later and we wouldn't notice), and could just apply for the whole section?

Reply:
We could do it but for performance reasons, I wanted to limit the scope of it. Is it a common thing for core classes to change?

Reply:
Very unlikely in this case for it to change, since it's component and considered stable API,  but I don't think it's issue for performance, as there wouldn't be much content in this section ever. 

I don't remember if the section has other content anymore (we used to have Stripe connect nudge at least), but would be good for CSS to apply to those too.

Reply:
Otherwise applying to `label` and `p` tags I think is safe for performance.

Reply:
Like the idea of using `label` instead of the core class. Thanks! Fix already pushed.